types/office-js-preview/index.d.ts

```diff
@@ -11946,6 +11948,14 @@ declare namespace Office {
          *
          * **Important**: In Outlook on the web, `setSignatureAsync` only works on messages.
          *
+         * **Important**: If your add-in implements the 
+         * {@link https://docs.microsoft.com/office/dev/add-ins/outlook/autolaunch | event-based activation feature using `LaunchEvent` in the manifest},
+         * and calls `setSignatureAsync` in the event handler, the following behaviors apply.
+         *
+         * - If the user is editing a meeting, the `setSignatureAsync` call returns an error.
+         * - When the user composes a new item (including reply or forward), the signature is set but doesn't modify the form. This means
```

Comment:
Do bullet lists need extra blank lines between items to render right in the docs? I'm remembering that for some reason, though I could be completely wrong. Please double check this renders fine in the ref docs after checkin.

Reply:
I think it does need the extra line.

Reply:
Updated